src/EA.Weee.Web/Areas/Scheme/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -107,7 +107,7 @@ public ActionResult Transfer(Guid organisationId)
                         NoteStatus.Rejected,
                         NoteStatus.Void,
                         NoteStatus.Returned
-                    }, SelectedComplianceYear(currentDate, manageEvidenceNoteViewModel), new List<NoteType>() { NoteType.Evidence }, false));
+                    }, SelectedComplianceYear(currentDate, manageEvidenceNoteViewModel), new List<NoteType>() { NoteType.Evidence, NoteType.Transfer }, false));
```

Comment:
Should be an existing test that checks the GetEvidenceNotesByOrganisationRequest is called with the expected values, will need to be updated to also check that the Transfer note type is passed